src/perf/impeller.md

```diff
@@ -0,0 +1,159 @@
+  are built upfront. Caching is explicit and under
+  the control of the engine.
+* **Instrumentable**: All graphics resources
+  (textures, buffers, pipeline state objects, and so on)
+  are tagged and labeled. Animations can be captured
+  and persisted to disk without affecting per-frame
+  rendering performance.
+* **Portable**: Not tied to a specific client rendering API.
+  Shaders are authored once and converted to
+  backend-specific formats as necessary.
+* **Uses modern graphics APIs effectively**:
+  Makes heavy use of (but doesn’t depend on)
+  features available in Modern APIs like Metal and Vulkan.
+* **Makes effective use of concurrency**:
+  Can distribute single-frame workloads across
+  multiple threads if necessary.
+
+## Availability
+
+Where can you currently use Impeller?
+
+### iOS
+
+Impeller is currently available for preview on iOS.
+You can enable it by passing `--enable-impeller`
+to the `flutter run` command or add the
+following under the top-level `<dict>` tag in an
+application's `Info.plist` file:
+
+```
+  <key>FLTEnableImpeller</key>
+  <true/>
+```
+
+The team continues to make improvements to iOS support,
+and if you encounter any slow frames or rendering
+fidelity issues with Impeller on iOS,
+please file an issue in the [GitHub tracker][file-issue]
+with a small reproducible test case, and include
+`[Impeller]` in the title.
+
+[file-issue]: {{site.github}}/flutter/flutter/issues/new/choose
+
+### Android
+
+Android support is under active development,
+and isn't yet ready for preview. On the master channel,
+it might not work at the tip-of-tree.
+However, you can get an indication of the direction
+that support is headed by experimenting with it in the
+3.7 stable release. To enable Impeller on Android,
+either pass `--enable-impeller` to the `flutter run`
+command or add the following to your
+`AndroidManifest.xml` file under the `<application>` tag:
+
+```
+  <meta-data
+    android:name="io.flutter.embedding.android.EnableImpeller"
+    android:value="true" />
+```
+
+## Status
+
+You can track the up-to-date status and future plans
+of Impeller by following along on the [GitHub project board][].
+However, here is a summary of recent progress.
+
+[GitHub project board]: {{site.github}}/orgs/flutter/projects/21
+
+The team is increasingly confident that Impeller on iOS
+will meet the rendering needs of nearly all
+existing Flutter apps. As of the 3.7 release,
+Impeller on iOS supports all but a few features.
+The remaining gaps are as follows:
+
+* Setting `Paint.isAntiAlias` to false is not yet implemented
+  ([docs]({{site.api}}/flutter/dart-ui/Paint/isAntiAlias.html)).
+  ([#104721]({{site.github}}/flutter/flutter/issues/104721))
+  ([#109956]({{site.github}}/flutter/flutter/issues/109956))
+* Support for the focal argument to `Gradient.radial`
+  isn't yet implemented ([docs]({{site.api}}/flutter/dart-ui/Gradient/Gradient.radial.html)).
+
+Recently fixed issues that aren't yet on the stable channel::
```

Comment:
```suggestion
Recently fixed issues that aren't yet on the stable channel:
```

Do we plan to update this section frequently? This may be difficult to keep up-to-date :)

Reply:
Good catch. thx!